requirements/constraints.txt

```diff
@@ -37,3 +37,5 @@ twine==1.15.0 ; python_version == "2.7"
 
 # django-model-utils==4.0.0 version dropped the support for Django 1.11 and Python 2.7 and dropped usage of six
 django-model-utils<4.0.0
+
+edx-rest-api-client<2.0.0
```

Comment:
I don't think this is needed anymore; version 3.0.2 is being used in edx-platform and should have fixed the compatibility problems with the last couple of releases.

Reply:
While using 3.0.2 version, some tests fail when ran collectively but pass individually. When downgrade this package to 1.9.2 everything become fine

Reply:
Resolved failures.